Removing head tags due to the way meteor automatically adds these to …

insaineyesay opened the pull request:
Removing head tags due to the way meteor automatically adds these to the template. This was causing a 'tried to load angular more than once error' when following the tutorial found here: https://www.meteor.com/tutorials/angular/creating-an-app

stubailo commented:
@Urigo do you have any insight into what is going on here?

Urigo commented:
@insaineyesay I believe this is because you haven't finished with step 2.4 yet and `ng-include` can't find `todos-list.ng.html` because it doesn't exist yet..
Please let me know if that solved your problem

insaineyesay commented:
Hey guys, 
Thanks for getting back to me. I tried that, and it definitely didn't fix it. I looked at the source of the app and it was loading the head 2x. So then I just went and followed a different tutorial here: http://angular-meteor.com/tutorials/angular1/bootstrapping and everything was working, and that's when i was sure it was the head tag in step 2.2, So I deleted everything, and started from 1. Ran into the same problem. Removed the head tag and then my todo list began to show (which also wasn't happening because of the error), and the error was gone. Just to be 300% sure I will try again, and get back to you. 

Thanks again

insaineyesay commented:
I can't even recreate the error now, must be noob user error. I'm not sure. #embarrassing. Thanks anyway for being so attentive. Awesome work guys.

stubailo commented:
Cool, glad it worked out!

Urigo commented:
Thanks for reporting @insaineyesay , keep sending your feedback!
